cabal-install/src/Distribution/Client/Outdated.hs

```diff
@@ -168,25 +172,33 @@ depsFromPkgDesc verbosity comp platform = do
 data ListOutdatedSettings = ListOutdatedSettings {
   -- | Should this package be ignored?
   listOutdatedIgnorePred :: PackageName -> Bool,
-  -- | Should major version bumps should be ignored for this package?
+  -- | Should major version bumps be ignored for this package?
```

Comment:
Question about these docs: do we mean minor version bumps here? Just judging by the name, i'd expect that to be minor rather than major.

Reply:
Just checked, weirdly enough, this comment is correct. What it does is to check whether there is any *minor* version update and ignores any major version bumps. 

E.g. in the case of `binary == 0.8.5.0`, `cabal outdated --minor` reports:
```
Outdated dependencies:
binary ==0.8.5.0 (latest: 0.8.8.0)
```

Reply:
I guess this may make sense: if minor updates are non-breaking bug-fixes, we don't want to miss any, while updating to breaking major versions is not something we want to be nagged about.